fitbenchmarking/fitbenchmarking
# Update comparison tables in the documentation

#### Description of Work
This PR updates the comparison tables in the docs ready for the next release. I think there is only one that needed updating, but if you find any more then please let me know.

Fixes #892

#### Testing Instructions

1. Build the docs `.\docs\make.bat html`. (pip install sphinx and m2r2 first)
2. Read through the documentation and check that all the tables are up-to-date.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
